packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -332,6 +332,55 @@ const proxy = resource.addProxy({
 });
 ```
 
+### Authorizers
+
+API Gateway [supports several different authorization types](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-to-api.html)
+that can be used for controlling access to your Rest APIs.
+
+The following CDK code allows access to an IAM user to a Rest API endpoint, via IAM policies:
+
+```ts
+const getBooks = books.addMethod('GET', new apigateway.HttpIntegration('http://amazon.com'), {
+  authorizationType: apigateway.AuthorizationType.IAM
+});
+
+iamUser.attachInlinePolicy(new iam.Policy(this, 'AllowBooks', {
```

Comment:
Feels like we can do better here. Something like:

```ts
getBooks.grantExecute(user);
```

Reply:
Perhaps, but that's not what I'm aiming to fix in this PR. This is what we have today and I'm just documenting it.

I've opened a separate issue to track this - https://github.com/aws/aws-cdk/issues/5198